### PR TITLE
py-{mizani,plotnine}: add packages.

### PR DESCRIPTION
**Pull request by olupton**
Dependencies taken from `setup.py` ([`py-mizani`](https://github.com/has2k1/mizani/blob/v0.7.3/setup.py#L39-L42), [`py-plotnine`](https://github.com/has2k1/plotnine/blob/v0.8.0/setup.py#L51-L60)).

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-mizani/package.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    """Mizani is a scales package for graphics. It is based on Hadley Wickham's
+    Scales package."""
+
+    pypi = "mizani/mizani-0.7.3.tar.gz"
+
+    version(
+        "0.7.3",
+        sha256="f521300bd29ca918fcd629bc8ab50fa04e41bdbe00f6bcf74055d3c6273770a4",
+    )
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-mizani/package.py:18]:
```python
depends_on('python@3.6:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-plotnine/package.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    based on ggplot2. The grammar allows users to compose plots by explicitly
+    mapping data to the visual objects that make up the plot."""
+
+    pypi = "plotnine/plotnine-0.8.0.tar.gz"
+
+    version(
+        "0.8.0",
+        sha256="39de59edcc28106761b65238647d0b1f6212ea7f3a78f8be0b846616db969276",
+    )
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-plotnine/package.py:19]:
```python
depends_on('python@3.6:', type=('build', 'run'))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-mizani/package.py (hunk 1 above)
Comment (olupton) [reply var/spack/repos/builtin/packages/py-mizani/package.py:18]: Done in 41dd0358424c3cd75acba4ee5226381a22fb14d3.

File: var/spack/repos/builtin/packages/py-plotnine/package.py (hunk 2 above)
Comment (olupton) [reply var/spack/repos/builtin/packages/py-plotnine/package.py:19]: Done in 41dd0358424c3cd75acba4ee5226381a22fb14d3.

**Review by adamjstewart — APPROVED** [review]

**Comment by olupton** [comment]
Looks like the GitLab CI needs to be restarted?

**Comment by alalazo** [comment]
@olupton Ci restarted :crossed_fingers:

**Pull request merged**